woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -155,6 +155,25 @@ def test_latlong_transform(latlong_df):
         pd.testing.assert_series_equal(actual, expected)
 
 
+def test_latlong_transform_empty_series(empty_latlong_df):
+    df_type = str(type(empty_latlong_df))
+    is_dask = "dask" in df_type
+    is_spark = "spark" in df_type
```

Comment:
Could we use `_is_dask_dataframe` and `_is_spark_dataframe`?

Reply:
what if I change the `if / elif` to `if "dask" in df_type / elif "spark" in df_type`? and remove these variables?

Reply:
Sure, that works too, but are there any drawbacks for using `if _is_dask_dataframe(empty_latlong_df) ... elif _is_spark_dataframe(empty_latlong_df)`?

Reply:
it would be fewer lines, but `_is_x_dataframe` will probably be the clearest in meaning